src/core/Item.java

```diff
@@ -0,0 +1,115 @@
+package core;
+
+import commands.Command;
+
+import java.io.*;
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Created by Marcelo Loayza on 17/05/2016.
+ * Modified by Marcela Barrionuevo  added functionality for location, type, sizeBytes, sizeKB, sizeMB, sizeTB, filePermissions and filePermissions on 18/05/2016
+ */
+public class Item {
+
+    private String name;
+    private String location;
+    private String type;
+    private double sizeBytes;
+    private double sizeKB;
+    private double sizeMB;
+    private double sizeGB;
+    private double sizeTB;
+    private Date lastModified;
+    private String filePermissions;
+
+    private Map<String, String> map;
+
+    public Item (java.io.File file) {
+
+        if (file.isFile()) {
+            int i = file.getName().lastIndexOf('.');
+            if (i > 0) {
+                name = file.getName().substring(0, i);
+                type = file.getName().substring(i + 1);
+            }
+        } else {
+            name = file.getName();
+            type = "";
+        }
+
+        lastModified = new Date(file.lastModified());
+        location = file.getPath();
+        sizeBytes = file.length();
+        sizeKB = (sizeBytes / 1024);
+        sizeMB = (sizeKB / 1024);
+        sizeGB = (sizeMB / 1024);
```

Comment:
all these file size logic looks like another method, remember methods should do one thing